pygsuite/sheets/cell.py

```diff
@@ -0,0 +1,127 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Tuple
+
+from pygsuite.common.style import Color, TextStyle
+
+
+@dataclass
+class CellFormat:
+
+    HORIZONTAL_ALIGNMENTS = ["LEFT", "CENTER", "RIGHT"]
```

Comment:
This is stylistic but these might be usefuls as enums rather than class properties.

This just helps with typehinting and validating arguments: instead of 

CellForm(vertical_alignment= 'TOP')

And checking that against the string list

They could do CellFormat(vertical_alignment= Alignment.TOP)